python/shotgun_desktop/startup.py

```diff
@@ -1098,22 +865,22 @@ def main(**kwargs):
     # it right away.
     shotgun_authenticator = None
 
-    # We have to import this in a separate try catch block because we'll be using
-    # shotgun_authentication in the following catch statements.
-    try:
-        # Reading user settings from disk.
-        settings = Settings(app_bootstrap)
-        settings.dump(logger)
-    
-        # get the shotgun authentication module.
-        shotgun_authentication = __import_shotgun_authentication_from_path(app_bootstrap)
-    except Exception, e:
-        __handle_unexpected_exception(splash, shotgun_authenticator, e, app_bootstrap)
-        return -1
+    # Do not import sgtk globally to avoid using the wrong sgtk once we bootstrap in
+    # the right config.
+    import sgtk
+    sgtk.LogManager().global_debug = True
+    app_bootstrap.add_logger_to_logfile(
+        sgtk.LogManager().root_logger
+    )
 
     # We have gui, websocket library and the authentication module, now do the rest.
     server = None
+    from sgtk import authentication
     try:
+        # Reading user settings from disk.
+        settings = Settings()
+        settings.dump(logger)
```

Comment:
is this displaying the settings in the logger? What does it look like? Do we need some header or pprint formatting or something? or is that all taken care of?